com.cyberbotics.Webots.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.cyberbotics.Webots</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>Webots</name>
+  <summary>Webots mobile robot simulator</summary>
+  <description>
+    <p>
+      Webots is an open source and multi-platform desktop application used to simulate robots. It provides a complete development environment to model, program and simulate robots.
+      It has been designed for a professional use, and it is widely used in industry, education and research. Cyberbotics Ltd. maintains Webots as its main product continuously since 1998.
+    </p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Cyberbotics Ltd.</p>
```

Comment:
according to line 5, this project is licensed under Apache-2.0 license, so this disclaimer is unecessary.